$OpenBSD: patch-scripts_Makefile,v 1.2 2010/12/29 10:26:35 jasper Exp $

- Set the correct path for the logging directory in the config file.
- Don't set the hostname of the machine the package was built on.
- Install the config files to share/examples/yaws/ and we'll
  @sample them later to the right location.

--- scripts/Makefile.orig	Sat Sep 25 14:41:31 2010
+++ scripts/Makefile	Wed Dec 29 11:23:38 2010
@@ -25,7 +25,7 @@ local_install:	../bin/yaws
 	cat yaws.conf.template | \
 		./Subst %yawsdir% "`pwd`/.."  | \
 		./Subst %logdir% $$HOME/yaws_logs  | \
-		./Subst %host% `hostname`  | \
+		./Subst %host% `FULL_HOSTNAME_HERE`  | \
 		./Subst %port% 8000  | \
 		./Subst %vardir% "$(VARDIR)"  | \
 		./Subst %docroot% `pwd`/../www  | \
@@ -41,9 +41,9 @@ yaws.conf:
 	@echo PREFIX is $(PREFIX)
 	cat yaws.conf.template | \
 		./Subst %yawsdir% "$(PREFIX)/lib/yaws"  | \
-		./Subst %logdir% "$(PREFIX)/var/log/yaws"  | \
+		./Subst %logdir% "/var/log/yaws"  | \
 		./Subst %vardir% "$(VARDIR)"  | \
-		./Subst %host% `hostname`  | \
+		./Subst %host% `FULL_HOSTNAME_HERE`  | \
 		./Subst %port% 80  | \
 		./Subst %docroot% "$(VARDIR)/yaws/www"  | \
 		./Subst %certdir% "$(ETCDIR)/yaws"  | \
@@ -62,9 +62,9 @@ install:	.install
 		then echo "Keeping old config file "; \
 		cp yaws.conf $(DESTDIR)$(ETCDIR)/yaws/yaws.conf.template; \
 	else \
-		echo "Installing $(DESTDIR)$(ETCDIR)/yaws/yaws.conf"; \
-		mkdir -p $(DESTDIR)$(ETCDIR)/yaws; \
-		cp yaws.conf $(DESTDIR)$(ETCDIR)/yaws/yaws.conf; \
+		echo "Installing $(DESTDIR)$(PREFIX)/share/examples/yaws/yaws.conf"; \
+		mkdir -p $(DESTDIR)$(PREFIX)/share/examples/yaws; \
+		cp yaws.conf $(DESTDIR)$(PREFIX)/share/examples/yaws/yaws.conf; \
 	fi
-	@cp ../ssl/yaws-cert.pem $(DESTDIR)$(ETCDIR)/yaws
-	@cp ../ssl/yaws-key.pem $(DESTDIR)$(ETCDIR)/yaws
+	@cp ../ssl/yaws-cert.pem $(DESTDIR)$(PREFIX)/share/examples/yaws
+	@cp ../ssl/yaws-key.pem $(DESTDIR)$(PREFIX)/share/examples/yaws
